olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -211,6 +211,20 @@ struct CORE_API RetrySettings {
    * @brief Evaluates responses to determine if the retry should be attempted.
    */
   RetryCondition retry_condition = DefaultRetryCondition;
+
+  /**
+   * @brief Set default expiration for any cache entry made by the according
+   * layer or catalog client.
+   *
+   * This setting only applies to the mutable cache and to the in-memory cache,
+   * but should not affect the protected cache as no entries are added to the
+   * protected cache in read-only mode. Set to std::chrono::seconds::max() to
+   * disable expiration. By default expiration is disabled.
+   *
+   * @note This only makes sense for data that has an expiration limit, e.g.
+   * volatile or versioned, and which is stored in cache.
+   */
+  std::chrono::seconds default_cache_expiration = std::chrono::seconds::max();
```

Comment:
It should be part of OlpClientSettings not part of RetrySettings.

Reply:
Nice :)